js/ui/components/clubhouse.js

```diff
@@ -53,6 +54,10 @@ const ClubhouseIface =
     <method name="hide"> \
       <arg type="u" direction="in" name="timestamp"/> \
     </method> \
+    <method name="getAnimationMetadata"> \
```

Comment:
This must have a counterpart in the clubhouse, but I don't see a PR opened for the clubhouse repo.

Reply:
This is not yet CR Ready.